packages/client/hmi-client/src/components/models/Model.vue

```diff
@@ -377,6 +383,26 @@ const toggleEditMode = () => {
 	}
 };
 
+// Cancel existing edits, currently this will:
+// - Resets changs to the model structure
+const cancelEdit = async () => {
+	isEditing.value = false;
+	if (!model.value) return;
+
+	// Convert petri net into a graph with raw input data
+	const g: IGraph<NodeData, EdgeData> = parsePetriNet2IGraph(model.value.content, {
```

Comment:
`graph` instead of `g` will be better.
Those mathematicians can't spell out variables, but we can 😄

Reply:
done